README.md

```diff
@@ -1,51 +1,199 @@
 # callkeep
 
-iOS CallKit and Android ConnectionService bindings for Flutter
+Showing incoming call notification/screen using iOS CallKit and Android Custom UI for Flutter
 
-## Getting Started
+## Native setup
 
-This project is a starting point for a Flutter
-[plug-in package](https://flutter.dev/developing-packages/),
-a specialized package that includes platform-specific implementation code for
-Android and/or iOS.
+flutter_callkeep requires the following permissions.
```

Comment:
That nice if we add images on readme.

Reply:
I'll try to improve the documentation later and add nice images 😅